### PR TITLE
fix subnetmask representation in config output for IPV6

### PR DESCRIPTION
**Pull request by chparimi**
Subnet mask was completely expanded for IPV6 cases. Agent is expecting compressed format.

### CODE REVIEW

**Comment by chparimi** [comment]
@Microsoft/omsagent-devs  for review

**Comment by chparimi** [comment]
pbuild succeeded

File: source/code/plugins/npmd_config_lib.rb [hunk 1]
```diff
@@ -57,9 +57,6 @@ class ::IPAddr
         def getNetMaskString
             _to_string(@mask_addr)
         end
-        def getIPString
```
Comment (robbiezhang) [inline source/code/plugins/npmd_config_lib.rb]: update the comment above

Comment (chparimi) [inline source/code/plugins/npmd_config_lib.rb]: Done.

**Pull request merged**